service/qos/service_level_controller.cc

```diff
@@ -260,17 +262,28 @@ void service_level_controller::update_from_distributed_data(std::chrono::duratio
     }
     if (_global_controller_db->distributed_data_update.available()) {
         sl_logger.info("update_from_distributed_data: starting configuration polling loop");
+        _logged_intervals = 0;
         _global_controller_db->distributed_data_update = repeat([this, interval] {
             return sleep_abortable<steady_clock_type>(std::chrono::duration_cast<steady_clock_type::duration>(interval),
                     _global_controller_db->dist_data_update_aborter).then_wrapped([this] (future<>&& f) {
                 try {
                     f.get();
-                    return update_service_levels_from_distributed_data().then_wrapped([] (future<>&& f){
+                    return update_service_levels_from_distributed_data().then_wrapped([this] (future<>&& f){
                         try {
                             f.get();
+                            _last_successful_config_update = std::chrono::steady_clock::now();
```

Comment:
nitpick: `seastar::lowres_clock` is cheaper to read than `std::chrono::steady_clock` and has a good enough resolution - effectively 10ms, but here we seem to operate on full seconds anyway: https://github.com/scylladb/seastar/blob/master/include/seastar/core/lowres_clock.hh . It's not very critical in this scenario, but it's good practice for the future.

Reply:
I'll make the change